xml/System.Security.Cryptography/HashAlgorithmName.xml

```diff
@@ -282,7 +282,15 @@ Due to collision problems with MD5 and SHA1, Microsoft recommends a security mod
       <Docs>
         <summary>Gets the underlying string representation of the algorithm name.</summary>
         <value>The string representation of the algorithm name, or <see langword="null" /> or <see cref="F:System.String.Empty" /> if no hash algorithm is available.</value>
-        <remarks>To be added.</remarks>
+        <remarks>
+          <format type="text/markdown"><![CDATA[
+
+## Remarks
+
+May be `null` or empty to indicate that no hash algorithm is applicable.
```

Comment:
`null` and empty have no special defined meaning on this type.  It's simply a struct, and therefore can have been initialized to default.

Remove this remark.

Reply:
This is just a repetition of the \<value> section, where it is better placed. This should be deleted, and the \<value> section modified to reflect the comment by @bartonjs.